src/main/java/edu/uz/dr/auctioneer/controller/AuctionsController.java

```diff
@@ -0,0 +1,52 @@
+package edu.uz.dr.auctioneer.controller;
+
+import edu.uz.dr.auctioneer.model.request.AuctionRequest;
+import edu.uz.dr.auctioneer.service.AuctionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.validation.Valid;
+
+@Controller
+public class AuctionsController {
+
+    private final AuctionService auctionService;
+
+    @Autowired
+    public AuctionsController(final AuctionService auctionService) {
+        this.auctionService = auctionService;
+
+        if (auctionService.getAll().isEmpty()) {
```

Comment:
No, to można pozmieniać, ale zastanawiałem się czy warto, bo nie wiem czy to w ogóle zostanie. Te dodawanie przykładowych aukcji to chyba tak tylko na chwilę, żeby za każdym razem nie trzeba było dodawać nowych. Czy planujemy zostawić coś takiego, żeby było na początku trochę przykładowych aukcji?